src/main/java/Interactors/GameCreation.java

```diff
@@ -0,0 +1,57 @@
+package Interactors;
+
+import Entities.Board;
+import Entities.Cell;
+import Entities.Player;
+import Entities.Property;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+public class GameCreation {
+
+    public void createGame() throws FileNotFoundException {
+        // This method is to create a brand-new game and initialize a new Board.
+        ArrayList<Player> players = new ArrayList<>();
+        Player player1 = new Player("red");
+        Player player2 = new Player("blue");
+        Player player3 = new Player("green");
+        Player player4 = new Player("yellow");
+        players.add(player1);
+        players.add(player2);
+        players.add(player3);
+        players.add(player4);
```

Comment:
Is this assuming there will be exactly 4 players in each game? I'm pretty sure it's 2-8 players. This implementation disregards the case where there can be more or less than 4 players. Also I'm assuming the players may want to choose their own names (but this is less important).

Reply:
Yeah we have to fix this. I think you should talk to the people handling the user input to coordinate how exactly the details on the Player instances will be made.